<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kpad</class>
 <widget class="QMainWindow" name="Kpad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>380</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewDocument"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_as_HTML"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_me"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuFormat_2">
     <property name="title">
      <string>Bullet Points</string>
     </property>
     <addaction name="actionDisc"/>
     <addaction name="actionCircle"/>
     <addaction name="actionSquare"/>
     <addaction name="actionDecimal"/>
     <addaction name="actionLowerAlpha"/>
     <addaction name="actionUpperAlpha"/>
     <addaction name="actionLowerRoman"/>
     <addaction name="actionUpperRoman"/>
    </widget>
    <addaction name="actionIncrease_Font"/>
    <addaction name="actionDecrease_Font"/>
    <addaction name="separator"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionLeft_Align"/>
    <addaction name="actionCenter_Align"/>
    <addaction name="actionRight_Align"/>
    <addaction name="separator"/>
    <addaction name="menuFormat_2"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuFormat"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewDocument">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew">
     <normalon>:/icons/icons/new-document.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen">
     <normaloff>:/icons/icons/open.png</normaloff>
     <normalon>:/icons/icons/open.png</normalon>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Shift+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave">
     <normalon>:/icons/icons/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs">
     <normalon>:/icons/icons/save-as.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit">
     <normaloff>:/icons/icons/exit.png</normaloff>
     <normalon>:/icons/icons/exit.png</normalon>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy">
     <normalon>:/icons/icons/copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste">
     <normalon>:/icons/icons/paste.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut">
     <normaloff>:/icons/icons/cut.png</normaloff>
     <normalon>:/icons/icons/cut.png</normalon>:/icons/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo">
     <normaloff>:/icons/icons/undo.png</normaloff>:/icons/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo">
     <normalon>:/icons/icons/redo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpFaq">
     <normalon>:/icons/icons/about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About me</string>
   </property>
  </action>
  <action name="actionIncrease_Font">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>Increase Font</string>
   </property>
  </action>
  <action name="actionDecrease_Font">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
   <property name="text">
    <string>Decrease Font</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextBold"/>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextItalic"/>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextUnderline"/>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionLeft_Align">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyLeft"/>
   </property>
   <property name="text">
    <string>Left Align</string>
   </property>
  </action>
  <action name="actionCenter_Align">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
   </property>
   <property name="text">
    <string>Center Align</string>
   </property>
  </action>
  <action name="actionRight_Align">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyRight"/>
   </property>
   <property name="text">
    <string>Right Align</string>
   </property>
  </action>
  <action name="actionSave_as_HTML">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPageSetup"/>
   </property>
   <property name="text">
    <string>Save as HTML</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionDisc">
   <property name="text">
    <string>Disc</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionDecimal">
   <property name="text">
    <string>Decimal</string>
   </property>
  </action>
  <action name="actionLowerAlpha">
   <property name="text">
    <string>LowerAlpha</string>
   </property>
  </action>
  <action name="actionUpperAlpha">
   <property name="text">
    <string>UpperAlpha</string>
   </property>
  </action>
  <action name="actionLowerRoman">
   <property name="text">
    <string>LowerRoman</string>
   </property>
  </action>
  <action name="actionUpperRoman">
   <property name="text">
    <string>UpperRoman</string>
   </property>
   <property name="toolTip">
    <string>UpperRoman</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
